gateway/src/test/java/io/camunda/zeebe/gateway/api/job/LongPollingActivateJobsTest.java

```diff
@@ -212,10 +214,11 @@ public void shouldCompleteFollowingRequestsAfterTimeout() {
     activateJobsStub.addAvailableJobs(TYPE, 1);
     brokerClient.notifyJobsAvailable(TYPE);
     handler.activateJobs(successRequest);
+    Awaitility.await().until(successRequest::isCompleted);
```

Comment:
:thought_balloon: Should we wait until it is completed? I think since we request 2 jobs, but only one job is available it will still wait for longPollingTimeout (=5s) to complete the request. So either we don't wait until it is completed (only verify that `onNext` is called), or ensure that the request can be completed fast by making 2 jobs available.

Reply:
Sorry.. This comment was meant to the changes in  `shouldUnblockRequestWhenJobsAvailable` :)

Reply:
Interesting point, but the test completes after about 200ms, so it doesn't look like it waits for this timeout.

<img width="415" alt="Screen Shot 2022-05-04 at 15 44 30" src="https://user-images.githubusercontent.com/3511026/166694223-fd4f6db5-b597-4eb0-a6c9-d11779399355.png">

I also tested it without awaiting until the request has been completed (and removing the assertion that onCompleted is called altogether), which is faster (somewhat unsurprisingly).
<img width="416" alt="Screen Shot 2022-05-04 at 15 45 39" src="https://user-images.githubusercontent.com/3511026/166694449-895d53d2-e42c-49c4-a3b7-1b0c9db0615d.png">

I do like the idea, but I'm just uncertain about our assumptions because of this. What do you think?

Reply:
Ok. Then ignore my comment. Looks like it completes the request because atleast one job was activated.